src/content-helper/editor-sidebar/title-suggestions/provider.ts

```diff
@@ -52,13 +52,13 @@ export class TitleSuggestionsProvider extends BaseProvider {
 	public async generateTitles( content: string, limit: number = 3, tone: ToneProp, persona: PersonaProp ): Promise<string[]> {
 		const response = this.fetch<string[]>( {
 			method: 'POST',
-			path: addQueryArgs( '/wp-parsely/v1/content-suggestions/suggest-headline', {
+			path: addQueryArgs( '/wp-parsely/v2/content-helper/title-suggestions/generate', {
 				limit,
 				tone: getToneLabel( tone ),
 				persona: getPersonaLabel( persona ),
 			} ),
 			data: {
-				content,
+				text: content,
```

Comment:
I kept the variable name as `content` purely because I think it's more semantic for the function to have a title and a content instead of a title and text. 

The API parameter is still text, though, to be consistent with the upstream API parameter.